src/App/Container.php

```diff
@@ -124,4 +130,19 @@ public function create(string $abstract, array $params = [])
 
         return new $abstract(...array_values($params));
     }
+
+    /**
+     * @param string $toolsBasePath
+     * @param string $magentoBasePath
+     * @return Container
+     * @throws ContainerException
+     */
+    public static function getInstance(string $toolsBasePath, string $magentoBasePath): Container
```

Comment:
Why do you need this method? I see only one usage in tests, there can be used `new Container()` as well.

Reply:
It dramatically improves the integration test performce

Reply:
So, it will make sense to make `__construct` private and change where we use `new Container()` to `Container::getInstance`

Reply:
Making constructor to be private does not give and advantage for generic scenario

Reply:
It's not right when the class has static `getInstance` method and can be created via `new` and as a result, it will be two different instances of the same class.

Reply:
If you need this only for tests better to move `getInstance` method to some factory in the "Test" namespace

Reply:
Added `Test/Integration/Container` with this method